CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 * @grafana/mimir-maintainers
 
 # Documentation.
-/docs/ @docs-metrics @grafana/mimir-maintainers
+/docs/ @grafana/docs-metrics @grafana/mimir-maintainers
```

Comment:
@replay, I am not sure what is accurate here:

* `@grafana/docs-metrics`
* `@docs-metrics`

Reply:
I'm not sure, but I noticed that in the project settings page under "collaborators and teams" I see `docs-squad` but I don't see `docs-metrics`

Reply:
Yes, looks like I do per https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax.

Reply:
> I'm not sure, but I noticed that in the project settings page under "collaborators and teams" I see `docs-squad` but I don't see `docs-metrics`

I dug around and created the new team. :)

Reply:
I then added the team, and added write permissions. The `CODEOWNERS` file is now valid.